src/backend/joanie/core/certifications/models.py

```diff
@@ -0,0 +1,74 @@
+import uuid
+
+from django.db import models
+
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from parler import models as parler_models
+
+from joanie.core.products import models as products_models
+
+
+class Certification(parler_models.TranslatableModel):
```

Comment:
I find the name not explicit. It's hard to understand the link between "Certificate" and "Certification". 
Why not name it "CertificateDefinition"?